examples/DotNetCore/EventHub/Controllers/HomeController.cs

```diff
@@ -0,0 +1,169 @@
+            _processorClient.StartProcessing();
+        }
+
+        public IActionResult Index()
+        {
+            ViewData["Messages"] = Settings?.Messages;
+            ViewData["FontColor"] = Settings?.FontColor;
+            ViewData["FontSize"] = Settings?.FontSize;
+            ViewData["BackgroundColor"] = Settings?.BackgroundColor;
+
+            return View();
+        }
+
+        public IActionResult Privacy()
+        {
+            return View();
+        }
+
+        [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
+        public IActionResult Error()
+        {
+            return View(new ErrorViewModel { RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier });
+        }
+
+        private void InitEventHubProcessor()
+        {
+            string eventHubConnectionString = _eventHubConnection.EventHubConnectionString;
+            string eventHubName = _eventHubConnection.EventHubName;
+            string blobStorageConnectionString = _eventHubConnection.BlobStorageConnectionString;
+            string blobContainerName = _eventHubConnection.BlobStorageContainerName;
+            string consumerGroup = _eventHubConnection.EventHubConsumerGroup;
+
+            //
+            // Check if the required configurations are available.
+            if (string.IsNullOrEmpty(eventHubConnectionString))
+            {
+                throw new ArgumentNullException(nameof(eventHubConnectionString));
+            }
+
+            if (string .IsNullOrEmpty(eventHubName))
+            {
+                throw new ArgumentNullException(nameof(eventHubName));
+            }
+
+            if (string.IsNullOrEmpty(blobStorageConnectionString))
+            {
+                throw new ArgumentNullException(nameof(blobStorageConnectionString));
+            }
+            
+            if (string.IsNullOrEmpty(blobContainerName))
+            {
+                throw new ArgumentNullException(nameof(blobContainerName));
+            }
+
+            //
+            // If no consumer group is provided, use default one.
+            if (string.IsNullOrEmpty(consumerGroup))
+            {
+                consumerGroup = EventHubConsumerClient.DefaultConsumerGroupName;
+            }
+
+            //
+            // Create blob storage client.
+            var storageClient = new BlobContainerClient(connectionString: blobStorageConnectionString,
+                                                        blobContainerName: blobContainerName);
+            
+            _processorClient = new EventProcessorClient(checkpointStore: storageClient,
+                                                        consumerGroup: consumerGroup,
+                                                        connectionString: eventHubConnectionString,
+                                                        eventHubName: eventHubName);
+        }
+
+        private string ReadFromConfigStore(string key, string label)
```

Comment:
>ReadFromConfigStore [](start = 23, length = 19)

This pattern doesn't look like intended usage of our configuration provider. If we need on demand lookups we should probably use the SDK directly. Then we can have a single client.